Neo4j.Driver/Neo4j.Driver/Internal/Connector/TcpSocketClient.cs

```diff
@@ -54,9 +54,10 @@ public async Task ConnectAsync(Uri uri, CancellationToken cancellationToken = de
             try
             {
                 var sslStream = CreateSecureStream(uri);
+                var clientCertificates = await GetClientCertificates().ConfigureAwait(false);
 
                 await sslStream
-                    .AuthenticateAsClientAsync(uri.Host, null, Tls12, false)
+                    .AuthenticateAsClientAsync(uri.Host, clientCertificates, Tls12, false)
```

Comment:
I would like, while we are here to add the ability for users to specify Tls13 (if they are using .net 5+)